run.py

```diff
@@ -59,57 +69,59 @@
 print("Extracting Features...")
 # Retrieve db rows and store as a dataframe
 df = dbHelper.getRowsAsDf("SELECT * from clips where url = '{:}' ORDER BY start ASC".format(vidUrl))
+# Drop null columns with features to be populated
+feature_list = ['mfcc', 'amplitude', 'pitch', 'word', 'subjectivity', 'polarity', 'pred_excitement', 'pred_highlight_rf', 'pred_highlight_nn']
+df.drop(columns=feature_list, axis=1, inplace=True)
+
+# Feature extraction: add video title
+df['video_title'] = videoDownloader.getVideoTitle(vidUrl)
+print(videoDownloader.getVideoTitle(vidUrl))
 
 # Feature extraction: mfcc values
 print("Extracting mfcc...")
 mfccVals = mfccExtractor.getMfcc(librosaPath, intervals)
 df['mfcc'] = mfccVals
```

Comment:
I think we should write to the db after every feature extraction. If something breaks down the line we'll still have the data saved and we won't have to re-process things

Reply:
I think we `should not` do this. This makes the assumption that everything is "prod" and we are writing to disk on the db (ergo $$$) no matter what.

Reply:
True. We could have a check before each db write then. I think saving the data to the db is a priority. It saves a lot of time if something breaks, we won't have to wait another 30 mins for speech2text to finish running, we could just comment it out since we already have that data.

Reply:
the `IF` something breaks case should be handled by a pipeline test file.

We will pollute our database if we have no way to test our pipeline without writing to DB.

Reply:
No offence but I don't think this is a good way of testing. There shouldn't be testing code mixed in with the actual script. I think if we want to do this the right way we should have a separate testing module that calls this script, provides an input, and validates the output. Or/and we could have unit tests for all the helpers/services

Reply:
Precisely! This is what I meant exactly what I meant by `pipeline test file`. We already have something like this below ... and we need the same for this `run.py`
https://github.com/mattlianje/climactic/blob/master/audio-analysis/test.py

Now imagine our test.py calls this run.py, and it starts making writes to db ... this is not desired.

Reply:
Ok makes sense, but I still think we should write after each feature extraction. So in my mind it would look some think like this. 
```
// extract 1 feature
if not test: write it column to db
```
instead of 
```
// extract all features
if not test: write all to db
```

Reply:
Ahh ok - now this makes sense! Modular is better than bulk. 
I will update this PR-boi accordingly and add a test.py.